include/lttng/event.h

```diff
@@ -142,6 +142,8 @@ enum lttng_event_context_type {
 	LTTNG_EVENT_CONTEXT_PREEMPTIBLE		= 17,
 	LTTNG_EVENT_CONTEXT_NEED_RESCHEDULE	= 18,
 	LTTNG_EVENT_CONTEXT_MIGRATABLE		= 19,
+	LTTNG_EVENT_CONTEXT_CALLSTACK_KERNEL	= 20,
```

Comment:
Add a comment specifying on which architecture this is supported.

Reply:
Done for LTTNG_EVENT_CONTEXT_CALLSTACK_USER